Modules/timemodule.c

```diff
@@ -56,12 +56,9 @@ static BOOL WINAPI PyCtrlHandler(DWORD dwCtrlType)
 static long main_thread;
 
 
-#if defined(__BORLANDC__)
-/* These overrides not needed for Win32 */
 #define timezone _timezone
```

Comment:
If these were previously only used if Borland was defined, and we don't care about supporting Borland - why not remove the defines as well?

Reply:
They are actually still necessary for MSVC. You can see that modern Python still has these `#defines`, with the only condition being that a modern version is in use (Although they were eventually moved from generic cross-platform names to Py-specific symbols https://github.com/python/cpython/commit/6673decfa0fb078f60587f5cb5e98460eea137c2)

These symbols are still present, though deprecated, https://learn.microsoft.com/en-us/cpp/c-runtime-library/daylight-dstbias-timezone-and-tzname?view=msvc-170